Employees/TempEmployeeJSON.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Employees.Entites;
+using System.Reflection;
+
+namespace Employees
+{
+    public class TempEmployeeJSON
+    {
+        public TempEmployee TempEmployee2 = new TempEmployee("Kelly", 45000, 5000, 70, 45);
```

Comment:
It is a testing variable so when I call the write to employee method I pass in that variable so it can write the values to a JSON file. Should I remove it?

Reply:
As per the comment for PermanenetEmployeeJSON.cs